CHANGELOG.md

```diff
@@ -13,7 +13,11 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 
 * Added support for adding two nullable columns.
 
-* Addded support for unsigned types in MySQL.
+* Added support for unsigned types in MySQL.
```

Comment:
Spell checking sounds like a hard problem to get right in an editor plugin. Just write one that allows you to quickly create Github issues "check speling" tagged as "easy for newcomers" instead! :trollface: